FakeNFT/Scenes /Statistic/StatisticViewController.swift

```diff
@@ -74,13 +74,13 @@ class StatisticViewController: UIViewController, StatisticViewProtocol {
         view.addSubview(loadingIndicator)
 
         NSLayoutConstraint.activate([
-            // Constraints для таблицы
+           
             tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 20), // 20 между Navigation Bar и таблицей
             tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 16),
             tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -16),
             tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
 
-            // Constraints для индикатора загрузки
+           
             loadingIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor),
             loadingIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor)
```

Comment:
Сейчас индикатор чуть ниже цента вьюшки крутится, можно добавить safeAreaLayoutGuide и будет ровненько

```suggestion
            loadingIndicator.centerYAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerYAnchor)
```